packages/mui-system/src/createStyled.js

```diff
@@ -245,3 +234,41 @@ export default function createStyled(input = {}) {
     return muiStyledResolver;
   };
 }
+
+export function applyStyled(props, componentName, overridesResolver) {
```

Comment:
Can we mark this as internal? We don't want people to accidentally stable upon it and use it.